packages/shared/src/adapters/NextMiddlewareAdapter.ts

```diff
@@ -8,7 +8,7 @@ export class NextRequestAdapter implements RequestAdapter {
   }
 
   setRequestCookie(name: string, value: string) {
-    this.req.cookies[name] = value;
+    this.req.cookies.set(name, value);
   }
```

Comment:
I guess this line should also apply to `[NextAdapter.ts](https://github.com/boredland/auth-helpers/blob/chore/next-middleware-upgrade/packages/shared/src/adapters/NextAdapter.ts)`

Reply:
sadly no, the cookies property in NextApiRequest is just an object:
```
cookies: Partial<{
        [key: string]: string;
    }>;
```